### DISCUSSION TITLE
Google Deeplink not working

### DISCUSSION THREAD

**Discussion by Rishabhstagein** [WeiCongcong/google-ads-deferred-deep-link#1] [Q&A]
Hey @WeiCongcong,
We are facing issue with this package, when we tried it using the CURL provided by google to test deeplinks it's working fine, but while testing on live using youtube ads, it's not listening to the deeplinks stored in shared pref, can you please help us resolve the issue?
We are using package version 1.0.0
Have setup all the permissions and settings required for Android Manisfest (Currently Testing in Android) and defined and initialise the class in the MainApp itself.
Listener got registered but didn't pick deeplink when installing app through it.

Please let me know if any other information is required to understand the issue.

<img width="607" alt="Screenshot 2024-12-20 at 10 40 23 AM" src="https://github.com/user-attachments/assets/9d62ca9e-65eb-42e0-9409-f76758be00db" />

**Comment by WeiCongcong** [comment]
Hi. @Rishabhstagein 
Do you mean the deeplinks were recorded in shared pref when using youtube ads, but can not be given by the plugin api?

**Reply by divyansh-coding** [reply]
Hello @WeiCongcong 
Deeplinks are not being recorded from youtube ads, irrespective of the plugin or direct native implementation. We want to know if there's any other way to do this or someone else who have faced this same issue.
